<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PvP</class>
 <widget class="QDialog" name="PvP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 93, 48);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(21, 160, 26, 0);
color: rgb(255, 255, 255);
border: none</string>
   </property>
   <property name="text">
    <string>Player 2 - Press 1 for rock, 2 for paper, 3 for scissors</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="RockButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(21, 160, 26);
border: none</string>
      </property>
      <property name="text">
       <string>Rock</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PaperButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(21, 160, 26);
border: none</string>
      </property>
      <property name="text">
       <string>Paper</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ScissorsButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(21, 160, 26);
border: none</string>
      </property>
      <property name="text">
       <string>Scissors</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(21, 160, 26, 0);
color: rgb(255, 255, 255);
border: none</string>
   </property>
   <property name="text">
    <string>Player 1 - Which do you choose?</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserVsCompLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(21, 160, 26, 0);
color: rgb(255, 255, 255);
border: none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="outcomeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(21, 160, 26, 0);
color: rgb(255, 255, 255);
border: none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>99</width>
     <height>49</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(21, 160, 26);
border: none</string>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(21, 160, 26);
border: none</string>
   </property>
   <property name="text">
    <string>Reset Wins</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ContinueLayout"/>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
